src/org/usfirst/frc/team694/robot/subsystems/CrabArm.java

```diff
@@ -30,7 +30,7 @@ public void acquire() {
     }
 
     public void flapOut() {
-        crabArmMotors.set(-1);
+        crabArmMotors.set(RobotMap.FLAP_OUT_SPEED);
```

Comment:
As stated before, keep the constant constant and negate this (-1 * FLAP_OUT_SPEED)